src/main/java/com/mixpanel/android/mpmetrics/MPDbAdapter.java

```diff
@@ -102,6 +102,13 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
             db.execSQL(PEOPLE_TIME_INDEX);
         }
 
+        public boolean belowMemThreshold() {
```

Comment:
I like this logic, but I wish that the MPDbAdapter wasn't responsible for knowing how much is too much? Perhaps it would be better to provide something like MPDbAdapter.getUsableSpace and MPDbAdapter.fileSize, and then have the comparison logic in AnalyticsMessages?